README.md

```diff
@@ -70,6 +82,8 @@ Used for optimizing images that come from backdrops and emblems.
  * `IMAGE_OPTIMIZE_KEY` - The key for API use.
  * `IMAGE_DOMAIN` - The domain for API Service
 
+Nginx listens at your local IP address, port 8080, e.g. http://localhost:8080
```

Comment:
Guess I missed this note - it doesn't really have anything to do with HaloDotApi section. Probably meant to be up by Docker stuff

Reply:
Fixed in #503